lib/internal/histogram.js

```diff
@@ -318,26 +327,36 @@ class RecordableHistogram extends Histogram {
   }
 }
 
-function internalHistogram(handle) {
+function ClonedHistogram(handle) {
```

Comment:
This ReflectConstruct can be removed as well

Reply:
Yeah, but since it is not exposed directly I didn't think that worth it, if not for spec compliance, this code could be removed (I think).